melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/tps/personsok/PersonSoekResponse.java

```diff
@@ -1,19 +1,12 @@
 package no.nav.melosys.eessi.service.tps.personsok;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class PersonSoekResponse {
```

Comment:
Jeg er forkjemper for "normalisering" med bl.a. ø->oe men andre steder i koden er det `PersonSok`